### PR TITLE
[DOC] Update FT.CREATE and FT.DROPINDEX temporary index deletion

### PR DESCRIPTION
**Pull request by nermiller**
Adds admonition about document deletion on temporary indexes to FT.CREATE and FT.DROPINDEX plus minor structural changes to other commands

### CODE REVIEW

**Comment by nermiller** [comment]
@slorello89, please take a look at FT.CREATE and FT.DROPINDEX notes. Thank you so much for your time!

File: docs/commands/ft.create.md [hunk 1]
```diff
@@ -97,23 +92,29 @@ is document attribute that you use as a binary safe payload string to the docume
...
 
-does not store term offsets for documents. It saves memory, but does not
-  allow exact searches or highlighting. It implies `NOHL`.
+does not store term offsets for documents. It saves memory, but does not allow exact searches or highlighting. It implies `NOHL`.
 </details>
 
 <details open>
 <summary><code>TEMPORARY</code></summary> 
 
 creates a lightweight temporary index that expires after a specified period of inactivity. The internal idle timer is reset whenever the index is searched or added to. Because such indexes are lightweight, you can create thousands of such indexes without negative performance implications and, therefore, you should consider using `SKIPINITIALSCAN` to avoid costly scanning.
```
Comment (oshadmi) [inline docs/commands/ft.create.md:107]: I suggest to also mention here the period is in seconds, although it is specified in the above syntax `[TEMPORARY seconds]`

File: docs/commands/ft.dropindex.md [hunk 2]
```diff
@@ -1,41 +1,43 @@
 ---
-syntax: 
+syntax: |
+  FT.DROPINDEX index 
+    [ DD]
```
Comment (oshadmi) [inline docs/commands/ft.dropindex.md:4]: Nitpick - consider a single line for simple commands or multi line - currently it is inconsistent
(some are spanning multi lines and some are not)
```suggestion
  FT.DROPINDEX index 
    [DD]
```

File: docs/commands/ft.sugadd.md [hunk 3]
```diff
@@ -1,21 +1,15 @@
 ---
-syntax: 
+syntax: |
+  FT.SUGADD key string score 
+    [ INCR]
```
Comment (oshadmi) [inline docs/commands/ft.sugadd.md:4]:
```suggestion
  FT.SUGADD key string score 
    [INCR] 
```

**Review by oshadmi — COMMENTED** [review]
Few minor comments

File: docs/commands/ft.create.md [hunk 4]
```diff
@@ -97,23 +92,29 @@ is document attribute that you use as a binary safe payload string to the docume
...
 
-When dropped, a temporary index does not delete the hashes as they may have been indexed in several indexes. Adding the `DD` flag deletes the hashes as well.
+{{% alert title="About using FT.DROPINDEX with temporary indexes" color="warning" %}}
+ 
+Historically, RediSearch used an FT.ADD command, which made a connection between the document and the index. Then, FT.DROP, also a hystoric command, deleted documents by default.
```
[the remark this answers is not in the document]
Comment (nermiller) [reply docs/commands/ft.create.md]: @slorello89, thanks for your feedback! I've updated the note. Let me know if we need to drop the historical context.

File: docs/commands/ft.create.md (hunk 1 above)
Comment (nermiller) [reply docs/commands/ft.create.md]: @oshadmi added seconds

File: docs/commands/ft.create.md [hunk 5]
```diff
@@ -97,23 +92,30 @@ is document attribute that you use as a binary safe payload string to the docume
...
 <summary><code>TEMPORARY</code></summary> 
 
-creates a lightweight temporary index that expires after a specified period of inactivity. The internal idle timer is reset whenever the index is searched or added to. Because such indexes are lightweight, you can create thousands of such indexes without negative performance implications and, therefore, you should consider using `SKIPINITIALSCAN` to avoid costly scanning.
+creates a lightweight temporary index that expires after a specified period of inactivity, in seconds. The internal idle timer is reset whenever the index is searched or added to. Because such indexes are lightweight, you can create thousands of such indexes without negative performance implications and, therefore, you should consider using `SKIPINITIALSCAN` to avoid costly scanning.
+
+{{% alert title="Warning" color="warning" %}}
+ 
+When temporary indexes expire, they drop all the records associated with them.
+`FT.DROPINDEX` was introduced with a default of not deleting docs and a `DD` flag that enforced deletion.
+However, for temporary indexes, you can expect the previous behavior, where documents are deleted along with the index.
```
Comment (slorello89) [inline docs/commands/ft.create.md:113]:
```suggestion
However, for temporary indexes, documents are deleted along with the index.
```

**Review by slorello89 — APPROVED** [review]
FT.CREATE / DROPINDEX bits LGTM 👍

**Review by oshadmi — APPROVED** [review]
LGTM 👍🏼

File: docs/commands/ft.dropindex.md [hunk 6]
```diff
@@ -1,41 +1,34 @@
...
 
 <details open>
 <summary><code>DD</code></summary>
 
 drop operation that, if set, deletes the actual document hashes.
-</details>
 
-<note><b>Notes:</b> 
+By default, FT.DROPINDEX does not delete the documents associated with the index. Adding the `DD` option deletes the documents as well. 
+If an index creation is still running (`FT.CREATE` is running asynchronously), only the document hashes that have already been indexed are deleted.
```
Comment (nermiller) [inline docs/commands/ft.dropindex.md:27]:
```suggestion
If an index creation is still running (`FT.CREATE` is running asynchronously), only the documents that have already been indexed are deleted. 
```

File: docs/commands/ft.dropindex.md [hunk 7]
```diff
@@ -1,41 +1,34 @@
...
 <details open>
 <summary><code>DD</code></summary>
 
 drop operation that, if set, deletes the actual document hashes.
-</details>
 
-<note><b>Notes:</b> 
+By default, FT.DROPINDEX does not delete the documents associated with the index. Adding the `DD` option deletes the documents as well. 
+If an index creation is still running (`FT.CREATE` is running asynchronously), only the document hashes that have already been indexed are deleted. 
+The document hashes left to be indexed remain in the database.
```
Comment (nermiller) [inline docs/commands/ft.dropindex.md:28]:
```suggestion
The documents left to be indexed remain in the database.
```

**Review by nermiller — COMMENTED** [review]
@oshadmi I'm applying your feedback now.

**Pull request merged**